README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
+
+<img alt="MacOSX Shell" src="https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/osx_shell.png" width=595>
+
+You can ask the computer to do many types of tasks via the shell (via the
+terminal). From the shell's command prompt (that you see in the terminal) you
+can move or rename files just like you would from a window environment.  You
 can _also_ get diagnostic information ("How is my computer?") or help your
 computer recover without requiring a reboot ("Hm, computer, time to kill that
 hung Chrome tab with the Twitch stream...").  You can ask the computer to run
-programs like `ruby my_awesome_program.rb` or even to launch an application!
-
-## Recognize a Shell Interface
+your own programs like `ruby my_awesome_program.rb` or even to launch an
+application!
 
-A shell interface looks like the following pictures below.
+When interacting with a shell via a terminal application, you'll be engaged in
+a cycle called the "prompt-evaluation-response cycle." By this, we mean that
+you will be given a command prompt, you will provide a command that will be
+evaluated by the shell with the help of the operating system and results will
+be printed for you. To trigger evaluation of your command you strike the
+`return` or `enter` keys.
 
-Typically you'll be given some text output and a command prompt. The prompt
-will receive commands. You tell the shell you're done typing and that it should
-consider your request by hitting `return` or `enter`. After receiving your
-command the shell will respond with the information you requested, or will
-signal that it did your bidding (say renaming a file), or it will print an
-error message. Then you'll be given another prompt and can do it again, and
-again, and again.... This cycle can be called a "prompt-evaluation-response
-cycle."
+### See Shells in the Wild
 
-Here's the Learn In-Browser IDE's shell. Since the shell environment is so
-important for us, whenever you open the IDE the command prompt is also shown.
-You will interacting with the shell when you run commands like `learn` and
-`learn submit`.
+Since the terminal environment is so important for us, whenever you open the
+IDE, the LearnIDE terminal is automatically shown.  You will interact with the
+shell through this terminal when you run commands like `learn` and `learn
+submit`.
 
 <img alt="Learn In-Browser IDE Shell" src="https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/learn_shell.png" width=595>
 
-Here's the Terminal application in MacOSX running a shell. You can launch it by
-navigating with the Finder to `Applications` &rarr; `Utilities` &rarr;
-`Terminal`.
+Here's the Terminal application in MacOSX running a terminal program. You can
+launch it by navigating with the Finder to `Applications` &rarr; `Utilities`
+&rarr; `Terminal`. There are other third-party terminal programs too. They all
+provide a way of using a shell. Liking one over the other is a matter of
+preference.
```

Comment:
Maybe add a line here like "Liking one over the other is a matter of preference, but they all provide similar functionality."